src/App.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] [OPTIONAL] Kindly as stated by the previous reviewer 😃 You have disabled ESLint checks on parts of your code. Please enable all ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.👌💯